src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -21,6 +21,62 @@ extern "C" {
 typedef halide_hexagon_remote_handle_t handle_t;
 typedef halide_hexagon_remote_buffer buffer;
 
+#define MAX_WORKER_THREADS 4
```

Comment:
Is there a way to query QURT to find out how many Hexagon cores there are, so this doesn't need to be hard-coded?